subprojects/android-test/junit-utils/src/main/java/com/avito/android/rule/SimpleRule.kt

```diff
@@ -10,15 +10,41 @@ abstract class SimpleRule : TestRule {
         object : Statement() {
             override fun evaluate() {
                 before()
```

Comment:
What will happen If before throws an Exception?

Reply:
This method in Rule executes prior to `@Before` methods and test itself.
If it fails, neither `@Before` nor test will be executed.
We will see an original and only one error.